ci/test/pipeline.template.yml

```diff
@@ -124,6 +124,15 @@ steps:
       queue: linux-x86_64
     coverage: skip
 
+  - id: lint-docs-catalog
+    label: Lint catalog docs
+    command: bin/ci-builder run stable ci/test/lint-docs-catalog.sh
+    inputs: [doc/user]
```

Comment:
Docs deviate when either the docs or the code change. I understand that we don't want to run this on every PR, but what about running it additionally in nightly?